ajax/libs/codemirror-spell-checker/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "codemirror-spell-checker",
+  "version": "1.1.2",
+  "filename": "spell-checker.min.js",
+  "description": "Dead-simple spell checking in CodeMirror",
+  "homepage": "https://nextstepwebs.github.io/codemirror-spell-checker/",
```

Comment:
@pvnr0082t Found the correct homepage: https://sparksuite.github.io/codemirror-spell-checker/